library/src/utils/tags.ts

```diff
@@ -13,7 +13,7 @@ export function argsMs(args: Set<string>) {
   return 0
 }
 
-export function argsHas(args: Set<string>, arg: string, defaultValue = false) {
-  if (!args) return defaultValue
-  return args.has(arg)
+export function tagHas(tags: Set<string>, tag: string, defaultValue = false) {
+  if (!tags) return defaultValue
+  return tags.has(tag.toLowerCase())
```

Comment:
@delaneyj this `toLowerCase()` seems pointless, since it refers to internal checks only.

Reply:
Yeah, good point.